backend/lib/edgehog/astarte/device/deployment_update.ex

```diff
@@ -0,0 +1,42 @@
+#
+# This file is part of Edgehog.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+defmodule Edgehog.Astarte.Device.DeploymentUpdate do
+  @moduledoc false
+
+  @behaviour Edgehog.Astarte.Device.DeploymentUpdate.Behaviour
+
+  alias Astarte.Client.AppEngine
+  alias Edgehog.Error.AstarteAPIError
+
+  @interface "io.edgehog.devicemanager.apps.DeploymentUpdate"
+
+  @impl Edgehog.Astarte.Device.DeploymentUpdate.Behaviour
+  def update(client, device_id, data) do
+    api_call = AppEngine.Devices.send_datastream(client, device_id, @interface, "/release", data)
```

Comment:
The interface changed: it now needs to be called on the `/deployment` endpoint (see [interfaces PR](https://github.com//pull/64/files#diff-0482701a7ded078199e43f0fcfc2291ca0b384dc5be49ac52969a3d3938e1e17))
```suggestion
    api_call = AppEngine.Devices.send_datastream(client, device_id, @interface, "/deployment", data)
```